src/app/hooks/usePopularMovies.ts

```diff
@@ -0,0 +1,16 @@
+import { PopularMovie } from '../../types';
+import useFetch from './useFetch';
+
+function usePopularMovies(): {
+  popularMovies: PopularMovie[] | null;
+  isLoading: boolean;
+  errorMessage: string | null;
+} {
+  const { data: popularMovies, isLoading, errorMessage } = useFetch<
+    PopularMovie[]
+  >('api/movies/popular');
```

Comment:
This could produce an error if the path or your URL is not /.
Imagine you are on /dashboard, which creates this API URL: /dashboard/api/movies/popular.
On / it is correct (/api/movies/popular).

```suggestion
  >('/api/movies/popular');
```

This makes sure, that it's always /api/movies/popular.